musicbot/bot.py

```diff
@@ -5307,7 +5313,8 @@ async def cmd_remove(
         author: discord.Member,
         permissions: PermissionGroup,
         player: MusicPlayer,
-        index: str = "",
+        position: str,
```

Comment:
This may need to have a default value to allow for the optional position argument.